### PR TITLE
Modified 3 files

### PR DESCRIPTION
**Pull request by nimrodmatan**
Committed by ops@velocitycareerlabs.com on behalf of nimrod.matan@velocitycareerlabs.com

### CODE REVIEW

File: toc.json [hunk 1]
```diff
@@ -478,6 +478,22 @@
         }
       ]
     },
+    {
+      "type": "group",
+      "title": "NCSBN Schema ",
```
Comment (nassan) [inline toc.json:483]: there is an extra space here, is that deliberate?

File: Layer 2/NCSBNMultiLicenseV1.0.json [hunk 2]
```diff
@@ -0,0 +1,913 @@
+{
```
Comment (nassan) [inline Layer 2/NCSBNMultiLicenseV1.0.json:1]: is it correct that the schema is v1.0 and the descriptor is v1.1?

File: Layer 2/NCSBNMultiLicenseV1.0.json [hunk 3]
```diff
@@ -0,0 +1,913 @@
...
+    "AdvancedPractice": {
+      "type": [
+        "object",
+        "null"
+      ],
+      "properties": {
+        "FocusSpecialty": {
+          "type": [
+            "string",
+            "null"
```
Comment (nassan) [inline Layer 2/NCSBNMultiLicenseV1.0.json:225]: we are deliberately allowing a null value, in addition to the property not being there? There are many examples of this. That will fulfill the requirement property. Meaning we are saying that StateAbbreviation is required, but a valid value for it is null? That is deliberate?

**Review by nassan — CHANGES_REQUESTED** [review]

File: Layer 2/NCSBNMultiLicenseV1.0.json (hunk 3 above)
Comment (nimrodmatan) [reply Layer 2/NCSBNMultiLicenseV1.0.json:225]: Yes. We are following NCSBN's schema conventions here.

**Review by nassan — APPROVED** [review]

**Pull request merged**